ejb-multi-server/install-domain.cli

```diff
@@ -47,6 +48,11 @@ batch
 # can not commit a transaction after executing the SLSB as the bean (with context) is destroyed before !!
 /profile=full/subsystem=ejb3:write-attribute(name=default-slsb-instance-pool, value=slsb-strict-max-pool)
 
+# configure default transaction workaround
```

Comment:
There is another workaround to this problem and that is getting the servers in domain mode to bind to "localhost" instead of 127.0.0.1. In that case, the client mappings entries will contain host=localhost and the problem will disappear. Might be worth considering.

Reply:
Instead of adding system properties, I'm going to update all interface definitions in host.xml and domain.xml using the install-domain.cli and replace the 127.0.0.1 values with localhost